build.sbt

```diff
@@ -113,7 +108,7 @@ lazy val extras = project
   .settings(sonatypeSettings: _*)
   .settings(
     name := "sbt-jmh-extras",
+    scalaVersion := "2.12.2",
```

Comment:
That's the only thing I am not sure about, since it's pure Java we may want to force it to target 1.6 to be able to use the plugin with a 1.6 JVM (using sbt 0.13.15)?